e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -101,6 +102,7 @@ def test_start_tensorboard_for_s3_experiment(tmp_path: Path, secrets: Dict[str,
             raise AssertionError(f"Did not find {SERVICE_READY} in output")
 
 
+@pytest.mark.skip()  # type: ignore
```

Comment:
blocking: why do you want to skip this?

Reply:
Because as far as I understand, the metrics are being generated by the GPU instance and saved to `/tmp` so when the CPU instance launches a tensorboard and looks for metrics in `/tmp/`, it can't find them.

Reply:
we should change this test to use `e2e_cpu`